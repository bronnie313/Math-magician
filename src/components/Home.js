const Home = () => (
  <div className="home">
    <h1>Welcome to our page!</h1>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut
      labore et dolore magna aliqua.Ut enim ad minim veniam, quis nostrud exercitation ullamco
      laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
      voluptate velit esse cillum dolore eu fugiat nulla pariayur. Excepteur sint occaecat cupidatat
      non proident, stunt in culpa qui officia deserunt molit anim id est laborum. Ipsu
    </p>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut
      labore et dolore magna aliqua.Ut enim ad minim veniam, quis nostrud exercitation ullamco
      laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
      voluptate velit esse cillum dolore eu fugiat nulla pariayur. Excepteur sint occaecat cupidatat
      non proident, stunt in culpa qui officia deserunt molit anim id est laborum. Ipsu
    </p>
  </div>
);

export default Home;
