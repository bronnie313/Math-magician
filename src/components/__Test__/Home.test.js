import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../Home';

describe('home component', () => {
  it('renders home component', () => {
    const tree = renderer.create(<Home />);
    expect(tree).toMatchSnapshot();
  });
});
