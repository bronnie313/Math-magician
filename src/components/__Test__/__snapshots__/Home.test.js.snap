// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home component renders home component 1`] = `
<div
  className="home"
>
  <h1>
    Welcome to our page!
  </h1>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariayur. Excepteur sint occaecat cupidatat non proident, stunt in culpa qui officia deserunt molit anim id est laborum. Ipsu
  </p>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariayur. Excepteur sint occaecat cupidatat non proident, stunt in culpa qui officia deserunt molit anim id est laborum. Ipsu
  </p>
</div>
`;
